cli/src/lib.rs

```diff
@@ -200,7 +214,7 @@ impl Iroha {
             config,
             instruction_judge,
             query_judge,
-            broker,
+            Broker::new(),
```

Comment:
broker can be implicit, i.e. removed from arguments?

Reply:
I'd like to do a separate pass on those functions if you don't mind. We have an issue to refactor them into builders.